### DISCUSSION TITLE
Duda sobre uso de código visto en clases

### DISCUSSION THREAD

**Discussion by gabo1243** [IIC2133-PUC/2024-1#183] [Tarea 3] [answered]
Hola, si llego a ocupar funciones/algoritmos que vimos en clase (exactamente igual), ¿Debo explicar a detalle la función o basta con decir que hace? Para el informe me refiero.

Gracias de antemano.

**Comment by 4n1t4** [comment, marked as answer]
debes detallar de igual forma, intenta que no sea exactamente la misma explicación que haces referencia. Dado que se debe transmitir un entendimiento  personal, por lo que se recomienda añadir observaciones u otro tipo de notas.
